testassets/Proto/chat.proto

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 The gRPC Authors
```

Comment:
doest it make sense to duplicate the .proto files between `examples/` and `testassets/` ?

Reply:
I had the same thought. But as @JamesNK mentioned, we expect the examples and testassets to diverge, see https://github.com/aspnet/grpc-dotnet/pull/6#discussion_r249913689.